From 4604ccd64bd9feb216dbb203671fcef65a24f315 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Tue, 13 Aug 2019 17:47:42 +0100
Subject: [PATCH 449/587] drm/vc4: Disable load tracking by default

The load tracking support appears to be broken which is causing
updates to be rejected (and duplicate mouse pointers!), so disable
it for now.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 drivers/gpu/drm/vc4/vc4_kms.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/vc4/vc4_kms.c b/drivers/gpu/drm/vc4/vc4_kms.c
index c52ae7fdcc36..e532041c83c1 100644
--- a/drivers/gpu/drm/vc4/vc4_kms.c
+++ b/drivers/gpu/drm/vc4/vc4_kms.c
@@ -517,10 +517,10 @@ int vc4_kms_load(struct drm_device *dev)
 	struct vc4_load_tracker_state *load_state;
 	int ret;
 
-	/* Start with the load tracker enabled. Can be disabled through the
+	/* Start with the load tracker disabled. Can be enabled through the
 	 * debugfs load_tracker file.
 	 */
-	vc4->load_tracker_enabled = true;
+	vc4->load_tracker_enabled = false;
 
 	sema_init(&vc4->async_modeset, 1);
 
-- 
2.17.1

